src/api-explorer/v4-0/SpendUser.swagger2.json

```diff
@@ -291,47 +298,47 @@
             "description": "The identifier for the employee within ADP, also known as the \"Employee File Number\"."
           }
         },
-        "description": "Adp settings for Employee"
+        "description": "A user's ADP settings"
```

Comment:
Is the ADP object properly wrapped in the `payroll` object?

Reply:
This Adp schema is for the `adp` object within the adp extension. The AdpExtension references this shema so I think the answer is yes?